include/nuttx/input/touchscreen.h

```diff
@@ -48,12 +50,32 @@
 
 /* Common TSC IOCTL commands */
 
-#define TSIOC_SETCALIB       _TSIOC(0x0001)  /* arg: Pointer to int calibration value */
-#define TSIOC_GETCALIB       _TSIOC(0x0002)  /* arg: Pointer to int calibration value */
-#define TSIOC_SETFREQUENCY   _TSIOC(0x0003)  /* arg: Pointer to uint32_t frequency value */
-#define TSIOC_GETFREQUENCY   _TSIOC(0x0004)  /* arg: Pointer to uint32_t frequency value */
-#define TSIOC_GETFWVERSION   _TSIOC(0x0005)  /* arg: Pointer to uint32_t firmware version value */
-#define TSIOC_ENABLEGESTURE  _TSIOC(0x0006)  /* arg: Pointer to int for enable gesture feature */
+#define TSIOC_SETCALIB       _TSIOC(0x0001) /* arg: Pointer to
```

Comment:
why we have two set calibration ioctl(TSIOC_SETCALIB/TSIOC_GETCALIB v.s. TSIOC_DOACALIB/TSIOC_CALDATA/TSIOC_USESCALED? Should we unify them.

Reply:
It's better to create a new patch to refine the ioctl name to distinguish the different operation.